src/app/models/notification.rb

```diff
@@ -0,0 +1,10 @@
+class Notification < ApplicationRecord
+  belongs_to :recipient, class_name: 'User', inverse_of: :notifications
+  belongs_to :actor, class_name: 'User', inverse_of: :notifications
```

Comment:
I don't get it? There's already a `has_many :notifications` in the user model, what do you mean?

Reply:
so what `user.notifications` returns? notifications were user is the recipient? or the actor?

Reply:
I assume it's where user is recipient because of the foreign key is assigned to recipient id
https://github.com/emad-elsaid/rubyfunctions/blob/5ab0c66f3760a1707b367f107e66cebc1a055a46/src/app/models/user.rb#L11-L12